serverless-now/env/sandbox.json

```diff
@@ -1,4 +1,5 @@
 {
+    "awsAccountId": "416527880812",
```

Comment:
Should this information be checked in? Just wondering if this should be kept somewhat private.

Reply:
Yup, already checked this with Nico a while back! :v: AWS Account ID, Cognito Client ID etc is all fine :v:
Good checking though, you obviously did a thorough review 🧙